semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/types/propertyaccessors/ModifierAccessors.scala

```diff
@@ -0,0 +1,30 @@
+package io.shiftleft.semanticcpg.language.types.propertyaccessors
+
+import gremlin.scala._
+import io.shiftleft.semanticcpg.language._
+import io.shiftleft.codepropertygraph.generated.{NodeKeys, NodeTypes, nodes}
+import io.shiftleft.codepropertygraph.generated.nodes.StoredNode
+import io.shiftleft.semanticcpg.language.types.expressions.generalizations.Modifier
+
+trait ModifierAccessors[T <: StoredNode] {
+  val raw: GremlinScala[T]
+
+  def hasModifier(modifier: String): Boolean =
```

Comment:
Please let this method filter directly instead of returning a boolean: `def hasModifier(modifier: String): Steps[T]